### ISSUE TITLE
Delete multiple subscriptions at once

### ISSUE THREAD

**Issue by Brawl345** [TeamNewPipe/NewPipe#3301]
#### Is your feature request related to a problem? Please describe it
I want to delete multiple subscriptions at once or delete them all at once.

#### Describe the solution you'd like
Maybe add some checkboxes to the subscriptions page with an option to select all and delete them?

#### How will you/everyone benefit from this feature?
I recently sorted out a lot of channels I don't follow anymore and I don't want to edit them in two places so I thought I could just delete them in the app and reimport but there is currently no function to do so.

**Comment by VivekThummar52** [comment]
Hey @Brawl345, i just want to delete/clear my Subscription list from the app, doesn't matter if I have to do it one by one, i want to clear it, right now when i long press on any subscription, it opens dialogue where there's option for Unsubscribe, not for removing from list, is there any way to remove from list, not Unsubscribe??

**Comment by Brawl345** [comment]
@VivekThummar52: What exactly do you want to do? If you want to still subscribe to all channels but just clear the "new videos from subscribed channels" overview, i don't think there is a way to do that. If you want to delete the subscriptions you need to indeed unsubscribe - this will only affect the app, not your YouTube account.

**Comment by opusforlife2** [comment]
Okay, I don't think I've seen this sort of Reddit-style chat here before.

@VivekThummar52 if you have a question, open a new issue and choose the 'question' template, or ask it in the Newpipe IRC, whose link you can get from the ReadMe. There is also the unofficial Newpipe subreddit. Asking for help inside a random Github issue like this is off-topic, and I'm marking it as such.

@Brawl345 Also marking yours, because it's a reply to an off-topic comment.

**Comment by opusforlife2** [comment]
Wait, @vivekthummar52 did you delete your comment? I set it to hide, but it vanished completely.

**Comment by VivekThummar52** [comment]
Yes, i deleted my comment because i also thought that it's off topic, also what i want is to show playlist from YouTube to NewPipe, which i have done already, so Thank you for your efforts and also Sorry for this.

**Comment by opusforlife2** [comment]
No problem! Glad you found a solution.

**Comment by githupper** [comment]
I need the batch unsubscribing function as well. 

I do not use NewPipe exclusively and sometimes I subscribe to new channels or  unsubscribe from dormant channels regularly on various platforms such as iPad, nvidia Shield, iPhone etc. 

Overtime Newpipe's subscription list becomes hard to manage as I need to browser the entire list to update it by removing or adding subscriptions.

 Overwriting with new YouTube export does not help either as it appends and not overwrite the current subscription list in Newpipe. 

As of right now Newpipe is my least used method of watching YouTube due to the subscription list always least accurate and relevant.

Please consider to allow mass deletion of subscription list or at least allow new import to overwrite current subscription list in Newpipe.

Thank you

**Comment by Shadow-302** [comment]
I second this. Such a useful feature.

**Comment by nair0xd** [comment]
I’ve made a pull request of the implementation of this feature : https://github.com/TeamNewPipe/NewPipe/pull/13833
